src/navigation/RotationControls.js

```diff
@@ -149,6 +151,15 @@ Potree.RotationControls = class RotationControls extends THREE.EventDispatcher{
 			return false;
 		}
 
+		const fixedRadiusDelta = Math.floor(this.radiusDelta * 1000) / 1000;
```

Comment:
what's 1000 ? magic number ? would it make more sense to give it a meaningful name ?

Reply:
So radius (zoom) delta is fading out over time. So if you are zooming the view the value get smaller over time - by smaller I mean it gets even values like 0.00000003.
Anyway, I forgot that I can use [toFixed()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) here to round the number. Let me update that.